lib/ddtrace/profiling/flush.rb

```diff
@@ -27,7 +28,8 @@ def initialize(*args)
         self.version = version || Datadog.configuration.version
         self.host = host || Datadog::Runtime::Socket.hostname
         self.language = language || Datadog::Runtime::Identity.lang
-        self.runtime = runtime || Datadog::Runtime::Identity.lang_interpreter
+        self.runtime_engine = runtime_engine || Datadog::Runtime::Identity.lang_engine
+        self.runtime_platform = runtime_platform || Datadog::Runtime::Identity.lang_platform
```

Comment:
We might want to report these in the tracer payload as well. Nothing to do in this PR, just a reminder for our future selves.

Reply:
Good idea. In the tracer payload I think we send them as "interpreter" which is both terms combined.